modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/other/APIScopeTestCase.java

```diff
@@ -239,20 +314,21 @@ public void destroy() throws Exception {
         }
 
         if (userManagementClient != null) {
-            userManagementClient.deleteUser(USER_JOHN);
-            userManagementClient.deleteRole(SUBSCRIBER_ROLE);
+
+            try {
+                userManagementClient.deleteUser(USER_JOHN);
+                userManagementClient.deleteRole(SUBSCRIBER_ROLE);
+            } catch (RemoteException e) {
+                handleException("Exception occured while removing the artifacts ", e);
+            } catch (UserAdminUserAdminException e) {
+                handleException("xception occured while removing the artifacts ", e);
```

Comment:
Currently our code is in frozen state due to release i will fix this once its released